editor/views/overview.py

```diff
@@ -47,29 +47,51 @@ def render_overview():
             on_change=handle_metadata_change,
             args=(MetadataEvent.DESCRIPTION, metadata, key),
         )
-
-        st.subheader(
-            f"{len(metadata.distribution)} File" + _plural(metadata.distribution)
-        )
-        st.subheader(
-            f"{len(metadata.record_sets)} Record Set" + _plural(metadata.distribution)
-        )
+        st.divider()
+        left, middle, right = st.columns([1, 1, 1])
+        fields = [
+            field
+            for field, value in dataclasses.asdict(metadata).items()
+            if value and field not in _NON_RELEVANT_METADATA
+        ]
+        left.metric("Number of metadata", len(fields))
+        middle.metric("Number of resources", len(metadata.distribution))
+        right.metric("Number of RecordSets", len(metadata.record_sets))
     with col2:
         user_started_editing = metadata.record_sets or metadata.distribution
         if user_started_editing:
-            st.subheader("Croissant File Validation")
+            warning = ""
             try:
                 issues = metadata.to_canonical().issues
                 if issues.errors:
-                    st.markdown("##### Errors:")
+                    warning += "**Errors**\n"
                     for error in issues.errors:
-                        st.write(error)
+                        warning += f"{error}\n"
                 if issues.warnings:
-                    st.markdown("##### Warnings:")
+                    warning += "**Warnings**\n"
                     for warning in issues.warnings:
-                        st.write(warning)
-                if not issues.errors and not issues.warnings:
-                    st.write("No validation issues detected!")
+                        warning += f"{warning}\n"
             except mlc.ValidationError as exception:
-                st.markdown("##### Errors:")
-                st.write(str(exception))
+                warning += "**Errors**\n"
+                warning += f"{str(exception)}\n"
+            if warning:
+                st.warning(warning, icon="⚠️")
+            else:
+                st.success("No validation issues detected!", icon="✅")
+        st.info(
+            """Croissant files are composed of three layers:
```

Comment:
Nit, Might be cleaner to keep those long text variables separated, where they are slightly easier to read?

Reply:
Done.